packages/common/src/authorization/projectMemberAbility.ts

```diff
@@ -42,6 +42,9 @@ export const projectMemberAbilities: Record<
         can('manage', 'ChangeCsvResults', {
             projectUuid: member.projectUuid,
         });
+        can('manage', 'ScheduledDeliveries', {
+            projectUuid: member.projectUuid,
```

Comment:
So now interactive viewers can edit any scheduled delivery, even the ones created by editors? doesn't feel right 🤔 @TuringLovesDeathMetal

Reply:
I couldn't find a linked Slack thread for this issue in Cloudy - where did this request come from? It'd be useful context

I've heard a version of this that viewers should be able to create their own deliveries - but editing ones that an admin has created doesn't seem right 🤔

Reply:
Yes, sorry, this was a request from Workday. I asked them about the permission to edit any scheduled delivery, and they said "yes", so I just added that to the permissions. 

Here's the very small info about this specific thing from our call: https://www.notion.so/lightdash/Workday-20dcba7760ca4ea09f999d24a6022330?pvs=4#d60f2b2653264220a8c5f1928c4f23fc (it's also in their shared feature doc). 

I initially also thought "no editing existing deliveries", but then the UX became more complicated and Workday said they didn't want that, so I updated the ticket.

Reply:
I updated this PR, you can view admin schedulers, but you can't update/delete them. This error is happening in the backend for now.